evaluators/text/BleuEvaluator/__init__.py

```diff
@@ -1,38 +1,38 @@
-from jina.executors.evaluators import BaseEvaluator
-from typing import Sequence, Any
 import nltk.translate.bleu_score as bleu
-import numpy as np
-from nltk import ngrams
-from collections import Counter
-
 
+from jina.executors.evaluators import BaseEvaluator
 
 class BleuEvaluator(BaseEvaluator):
     """
     :class:`BleuEvaluator`Bilingual Evaluation Understudy Score. 
-    Evaluates the generated sentence (hypothesis) against a reference sentence. 
+    Evaluates the generated sentence (actual) against a desired sentence. 
     It will use the Bleu on NLTK package.
     A perfect match will score 1.0 and a perfect unmatch will score 0.0
     """
     
     
-    def get_nltk_bleu_score(self, hypothesis, reference):
-        return bleu.sentence_bleu(reference, hypothesis)
+    def get_nltk_bleu_score(self, desired, actual):
+        return bleu.sentence_bleu(desired, actual)
 
 
     def evaluate(self,
```

Comment:
change the order, it is `actual` and then `desired`

Reply:
Where you mean is it in that order? in the nltk lib?

Reply:
The interface of the BaseEvaluator, and this is what the driver will provide.

Reply:
Ahhh right, missed it